bitnami/kafka/templates/NOTES.txt

```diff
@@ -78,6 +79,16 @@ IPs for Kafka brokers. To complete your deployment follow the steps below:
 
 ---------------------------------------------------------------------------------------------
 {{- end }}
+{{- if and .Values.externalAccess.enabled (not .Values.externalAccess.autoDiscovery.enabled) (gt $loadBalancerNameListLength 0) (not (eq $replicaCount $loadBalancerNameListLength)) (eq .Values.externalAccess.service.type "LoadBalancer") }}
+---------------------------------------------------------------------------------------------
+ WARNING
+
+    If "externalAccess.service.loadBalancerNames" is specified it should have the same number
+    of names as replicas has been deployed. This deployment could be incomplete, some of the
+    brokers could be not reachable for outside of K8S
+
+---------------------------------------------------------------------------------------------
+{{- end}}
```

Comment:
This shouldn't be handled here but using a "validateValues" helper, check the one below as reference:
- https://github.com/bitnami/charts/blob/master/bitnami/kafka/templates/_helpers.tpl#L359

Reply:
Initially I didn't include it because that implies a failure in the installation and for a similar `externalAccess.service.loadBalancerIPs` validation the approach was warn the user, I had my doubts and I decided to use the same approach as in `externalAccess.service.loadBalancerIPs` validation. But I agree, raise a failure could be more accurate in this case